llvm trunk version

trws opened the pull request:
Adding a trunk version to the llvm package.  This has all the features
and requirements of the others, with the additional caveat that the llvm
project makes no guarantee that trunk on all repositories together will
necessarily make a working compiler.  It has been tested, and worked
with a version today, but not yesterday, so if you test keep that in
mind.

alalazo commented:
@trws Not at all. I saw you added the `trunk` version using the same data structures that the released versions are using. I also added the same thing naming it `develop` and in a more brute-force manner. I was just suggesting removing the version I added as yours fits better with how the package is coded...

trws commented:
Ahhhh, got it. Thanks!


________________________________
From: Massimiliano Culpo
Sent: Sunday, February 21, 2016 1:36:01 PM
To: LLNL/spack
Cc: Scogland, Thomas Richard William
Subject: Re: [spack] llvm trunk version (#450)


@trws<https://github.com/trws> Not at all. I saw you added the trunk version using the same data structures that the released versions are using. I also added the same thing naming it develop and in a more brute-force manner. I was just suggesting removing the version I added as yours fits better with how the package is coded...

—
Reply to this email directly or view it on GitHub<https://github.com/LLNL/spack/pull/450#issuecomment-186923323>.
